golang/x/relay/client/cli/tx.go

```diff
@@ -23,31 +25,62 @@ func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
 	}
 
 	relayTxCmd.AddCommand(client.PostCommands(
-		GetCmdSetLink(cdc),
+		GetCmdIngestHeaderChain(cdc),
 	)...)
 
 	return relayTxCmd
 }
 
-// GetCmdSetLink is the CLI command for sending a SetLink transaction
-func GetCmdSetLink(cdc *codec.Codec) *cobra.Command {
+// GetCmdIngestHeaderChain creates a CLI command to ingest a header chain
+func GetCmdIngestHeaderChain(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "setlink [hex header] [signer address]",
+		Use:   "ingestheaders <json list of headers>",
```

Comment:
`ingestheaderchain`

Reply:
in the interest of being shorter to type in a cli, might leave it like this. next line needs fixing though

Reply:
My main concern is there is an `ingestHeaderChain` function and an `ingestHeaders` function.  So calling this `ingestHeaders` might be confusing

Reply:
eh, `ingestHeaders` is internal only. it's never part of the outside interface

Reply:
Ok, that's fine then